lib/generators/hyrax/templates/config/hyrax.rb

```diff
@@ -74,6 +74,10 @@
   # Path to the file derivatives creation tool
   # config.libreoffice_path = "soffice"
 
+  # Option to enable/disable full text extraction from PDFs
+  # Default is false, set to true to skip extraction
+  config.extract_full_text = true
```

Comment:
In the template typically we comment out the default options just to show what they are.  The default should be set in `Configuration`